esx_service/utils/auth_data.py

```diff
@@ -359,15 +356,16 @@ def create_tables(self):
                 '''
                     CREATE TABLE tenants(
                     -- uuid for the tenant, which is generated by create_tenant() API
-                    id TEXT PRIMARY KEY NOT NULL,
+                    id TEXT NOT NULL,
                     -- name of the tenant, which is specified by user when creating the tenant
                     -- this field can be changed later by using set_name() API
-                    name TEXT,
+                    name TEXT NOT NULL,
                     -- brief description of the tenant, which is specified by user when creating the tenant
                     -- this field can be changed laster by using set_description API
                     description TEXT,
                     -- not use currently
-                    default_datastore TEXT
+                    default_datastore TEXT,
+                    PRIMARY KEY(id, name)
```

Comment:
why do we need a key with both id, name  ? does it mean the key changes when name changes ? 
I thought ID would be the key and name would be just a text field (unique is good, but just text, not key) ?